src/deploy/osp_deployer/director.py

```diff
@@ -1289,31 +1289,24 @@ def setup_sriov_nic_configuration(self):
         sriov_vfs_setting = []
         sriov_map_setting = []
         sriov_pci_passthrough = []
-        physical_network = "physint"
-        physical_network1 = "tenant1"
-        physical_network2 = "tenant2"
-        check = 1
-        for interface in sriov_interfaces:
+        physical_network = ["physint", "physint1"]
+        check = 0
+        for interface in sriov_interfaces:   # Do it for only first two interfaces, if smart nic is enabled. otherwise
```

Comment:
53 additional pep8 compliance violations have been introduced with this PR to director.py.  I recommend these be addressed.  The following were present prior to the PR and is provided here as an example of how to run pep8 and to view the errors.  

pep8 director.py #where Ennn=Error and Wnnn=Warnings. 

director.py:167:16: E111 indentation is not a multiple of four
director.py:167:80: E501 line too long (82 > 79 characters)
director.py:169:16: E111 indentation is not a multiple of four
director.py:171:16: E111 indentation is not a multiple of four
director.py:171:80: E501 line too long (81 > 79 characters)
director.py:173:16: E111 indentation is not a multiple of four
director.py:175:1: W293 blank line contains whitespace
director.py:177:5: E303 too many blank lines (3)
director.py:442:46: W605 invalid escape sequence '\$'
director.py:967:38: W605 invalid escape sequence '\ '
director.py:985:30: W605 invalid escape sequence '\ '